### ISSUE TITLE
[Bug]  AutomationProperties.IsInAccessibleTreeProperty setting keyboard focus

### ISSUE THREAD

**Issue by rwaldenfraedom** [xamarin/Xamarin.Forms#11047]
### Description
When using `AutomationProperties.IsInAccessibleTreeProperty` in XF, it sets both ImportantForAccessibility (Talkback) & Focausable (Keyboard navigation) on the underlying Android View.

This means that when using a keyboard and tabbing though the screen, extra view are been focused on rather than just the Interactive views (button, TextEdit etc....)

As we have complex business requirements/ UI, certain elements are grouped together (multiple labels, labels and images etc...) for A11y purposed, these groups should be read out as one item. Using `AutomationProperties.IsInAccessibleTreeProperty` allows us to do that, but it also make these groups focusable when using keyboard navigation. (Please see screen shot)

Link to where _**an**_ instance of the bug may originate from: https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/FastRenderers/AutomationPropertiesProvider.cs#L98

### Steps to Reproduce

1. Create a vertical stack with two elements, a label and a Button.
2. Set AutomationProperties.IsInAccessibleTreeProperty = 'true' on both elements
3. Run app
4. Using keyboard, tab through the elements. 

### Expected Behavior
Out of the two elements created, only the _**button**_ should be focused on.

### Actual Behavior
By pressing tab, the label is focused on, then the button.

### Basic Information

- Version with issue: 4.5 -  but mostly likely any existing issues before 4.5
- Affected Devices: Android

### Screenshots

![KeyboardFocus](https://user-images.githubusercontent.com/40189228/84638368-020c7400-af4b-11ea-9765-a5d5925d4d27.png)

_**(NOTE: our focus colour is current incorrect, but if you look carefully, the amount  262.75 NZD is greyed - the grey overlay is from using the keyboard navigation focusing on the amount)**_

### Workaround

We have created own effect and attached property to replace `AutomationProperties.IsInAccessibleTreeProperty` The effect only sets the View.ImportantForAccessibility in Android.

**Comment by arevellfraedom** [comment]
It is problably important to note that the "keyboard" mentioned above is an external USB/bluetooth keyboard and does not relate to on-screen keyboards

**Comment by arevellfraedom** [comment]
@PureWeen This is an example of the things we are discovering about Xamarin Forms in relation to support for external Android keyboards

**Comment by rachelkang** [comment]
Hi, @rwaldenfraedom  - if you're still having issues, I recommend checking out the [`SemanticInclusion`](https://github.com/xamarin/XamarinCommunityToolkit/blob/000acd0f81ac65772b72d06de71ac763465d959b/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Semantic/SemanticInclusion.shared.cs#L5-L11) property from XamarinCommunityToolkit as this may prove to be a better alternative for your use case. Here is an [example ](https://github.com/xamarin/XamarinCommunityToolkit/blob/000acd0f81ac65772b72d06de71ac763465d959b/samples/XCT.Sample/Pages/Effects/SemanticEffectPage.xaml#L231-L233)for how it can be used

If you haven't already, I also recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: `Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});`

Let us know how that works for you!
